src/mds/Server.cc

```diff
@@ -5748,9 +5748,12 @@ int Server::parse_quota_vxattr(string name, string value, quota_info_t *quota)
           return r;
       }
     } else if (name == "quota.max_bytes") {
-      int64_t q = boost::lexical_cast<int64_t>(value);
-      if (q < 0)
+      string cast_err;
+      int64_t q = strict_iec_cast<int64_t>(value, &cast_err);
+      if(!cast_err.empty()) {
+        derr << cast_err << dendl;
```

Comment:
This will pollute/spam the log with a buggy client application. Instead:

```suggestion
        dout(10) << __func__ << ":  failed to parse quota.max_bytes: " << cast_err << dendl;
```

Reply:
i've fixed it, PTAL. Thanks!